examples/keras_imagenet_resnet50.py

```diff
@@ -110,32 +119,34 @@
     # TensorBoard, or other metrics-based callbacks.
     hvd.callbacks.MetricAverageCallback(),
 
-    # Horovod: using `lr = 1.0 * hvd.size()` from the very beginning leads to worse final
-    # accuracy. Scale the learning rate `lr = 1.0` ---> `lr = 1.0 * hvd.size()` during
+    # Horovod: using `lr = 1.0 * hvd.size()` from the very beginning leads to
+    # worse final accuracy.
+    # Scale the learning rate `lr = 1.0` ---> `lr = 1.0 * hvd.size()` during
     # the first five epochs. See https://arxiv.org/abs/1706.02677 for details.
-    hvd.callbacks.LearningRateWarmupCallback(warmup_epochs=warmup_epochs, verbose=verbose),
-
-    # Horovod: after the warmup reduce learning rate by 10 on the 30th, 60th and 80th epochs.
-    hvd.callbacks.LearningRateScheduleCallback(start_epoch=warmup_epochs, end_epoch=30, multiplier=1.),
-    hvd.callbacks.LearningRateScheduleCallback(start_epoch=30, end_epoch=60, multiplier=1e-1),
-    hvd.callbacks.LearningRateScheduleCallback(start_epoch=60, end_epoch=80, multiplier=1e-2),
+    hvd.callbacks.LearningRateWarmupCallback(warmup_epochs=warmup_epochs,
+                                             verbose=verbose),
+
+    # Horovod: after the warmup reduce learning rate by 10 on the 30th, 60th
+    # and 80th epochs.
+    hvd.callbacks.LearningRateScheduleCallback(start_epoch=warmup_epochs,
```

Comment:
We could keep this file at line limit of 110-120 chars as it is right now -- much fewer line breaks.

Reply:
Done.